products/green_fast_track/seeds/variables.csv

```diff
@@ -1,39 +1,55 @@
+e_des_hazmat,(E) Designations - HazMat,e_designations_hazmat,dcp_edesignation_csv,FALSE,0,HazMat Code = 1
+natural_resources,State Regulated Freshwater Wetlands,freshwater_wetlands,nysdec_freshwater_wetlands,FALSE,0,
+natural_resources,State Regulated Tidal Wetlands,tidal_wetlands,nysdec_tidal_wetlands,FALSE,0,
+natural_resources,Priority Waterbodies - Lakes,priority_waterbodies_lakes,nysdec_priority_lakes,FALSE,0,
+natural_resources,Priority Waterbodies - Estuaries,priority_waterbodies_estuaries,nysdec_priority_estuaries,FALSE,0,
+natural_resources,Priority Waterbodies - Streams,priority_waterbodies_streams,nysdec_priority_streams,FALSE,0,
+natural_resources,Beaches,beaches,dcp_beaches,FALSE,0,
+natural_resources,Significant Natural Communities,natural_heritage_communities,nysdec_natural_heritage_communities,FALSE,0,
+natural_resources,National Wetlands,usfws_wetlands,usfws_nyc_wetlands,FALSE,0,
+natural_resources,Recognized Ecological Complexes,rec,dcp_wrp_rec,TRUE,0,
+natural_resources,Special Natural Waterfront Areas,snwa,dcp_wrp_snwa,FALSE,0,
+natural_resources,NYC Parks Forever Wild,forever_wild,dpr_forever_wild,FALSE,0,
+wetland_checkzone,State Regulated Freshwater Wetlands - Checkzone,wetlands_checkzones,nysdec_freshwater_wetlands_checkzones,FALSE,0,
+archaeological_area,State Archeological Buffer Areas,archaeological_areas,nysshpo_archaeological_buffer_areas,FALSE,0,
+historic_districts,NYC Historic Buildings,nyc_historic_districts,lpc_historic_district_areas,FALSE,0,
+historic_districts,State Historic Buildings,nys_historic_districts,nysshpo_historic_buildings_polygons,FALSE,0,
+historic_districts,Scenic Landmarks,scenic_landmarks,lpc_scenic_landmarks,FALSE,0,
+historic_districts,Federal Historic Places,us_historic_places,us_historic_places,TRUE,0,
+historic_resources,NYC Historic Buildings,nyc_historic_buildings,lpc_landmarks,TRUE,0,
+historic_resources,State Historic Buildings,nys_historic_buildings,nysshpo_historic_buildings_points,TRUE,0,
+historic_resources_adj,City Historic Buildings,nyc_historic_buildings,lpc_landmarks,TRUE,90,
+historic_resources_adj,State Historic Buildings,nys_historic_buildings,nysshpo_historic_buildings_points,TRUE,90,
+shadow_open_spaces,DPR Parks properties,nyc_parks_properties,dpr_parksproperties,FALSE,200,
+shadow_open_spaces,Schoolyard-to-playground,schoolyard_to_playgrounds,dpr_schoolyard_to_playgrounds,FALSE,200,
+shadow_open_spaces,POPS,pops,dcp_pops,TRUE,200,
+shadow_open_spaces,Waterfront Public Access Areas,waterfront_access_wpaa,dcp_waterfront_access_map_wpaa,FALSE,200,
+shadow_open_spaces,Publicly Owned Waterfront,waterfront_access_pow,dcp_waterfront_access_map_pow,FALSE,200,
+shadow_open_spaces,NYS Parks Property,nys_parks_properties,nysparks_parks_polygons,FALSE,200,
+shadow_open_spaces,Federal Parks property,us_parks_properties,usnps_parks,FALSE,200,
+shadow_nat_resources,NYC Parks Forever Wild,forever_wild,dcp_beaches,FALSE,200,
```

Comment:
I see what you mean here - but these buffer values (200) are never used, are they? So given that shadow_nat_resources has a different buffer than natural_resources, the current shadow_nat_resources column is wrong

Reply:
Since it isn't actually buffered

Reply:
So these rows I think should just be dropped - as written, the join from the flags table to this creates duplicate rows independent of desired buffer distance. And for now, while we don't have variables that map explicitly to multiple questions, maybe keep the unique constraint on this table

Reply:
very true, this now generates incorrect `shadow_nat_resources` results

> we don't have variables that map explicitly to multiple questions

`nys_historic_buildings` and `nyc_historic_districts ` do. and these natural resources datasets do.

I was gonna say I'll restore `int_buffers__natural_resource_shadows` to it's former glory, but I think it'd be better to:
- keep these seed changes so that `variables` is unique on [`flag_id_field_name`, `variable_type`]
- let some outputs be incorrect for now
- implement the use of this new buffer column to fix those incorrect outputs

Reply:
I definitely think long-term this file and the way it's written make more sense, and it's a more elegant approach than the custom shadows buffer. But it's going to take some decent downstream changes to make it work: either generating multiple buffers from the same inputs programatically, which is something that we've talked about potentially being difficult in dbt, or have two staging tables for each of these datasets, or not actually have those buffers as their own models (which seems less than ideal).

Regardless, given that complexity, I don't think this should be merged to main without knowing how we're going to fix the issues it creates.

Reply:
happy to take a shot in this PR at generating flag-level intermediate models (programmatically or not) to align with this file

the end result would be:
- correct buffers and outputs
- use of this new approach
- would help with updating the `source__` export to meet Jack's needs for the map in the app: one layer per flag

Reply:
@fvankrieken the new commit most related to this is https://github.com/NYCPlanning/data-engineering/pull/826/commits/a3b9cae04d66778bc9e351733f91bbcb2e44a80c